yarn.lock

```diff
@@ -5570,6 +5570,11 @@ minimist@^1.1.0, minimist@^1.1.1, minimist@^1.1.3, minimist@^1.2.0:
   resolved "https://registry.yarnpkg.com/minimist/-/minimist-1.2.0.tgz#a35008b20f41383eec1fb914f4cd5df79a264284"
   integrity sha1-o1AIsg9BOD7sH7kU9M1d95omQoQ=
```

Comment:
I think you're also going to have to delete these older entries for `minimist` and `mkdirp` in here and re-run `yarn` to get it to update the other transitive dependencies, otherwise it's not going to clear the vulnerability warning. It looks like the semver constraints will allow them to use 1.2.5 and 0.5.3, too.

Reply:
> otherwise it's not going to clear the vulnerability warning

actually, the warnings may be computed on a `yarn.lock`-less install, so it might be fine, but you should still delete those so we don't have old and busted unnecessary versions hanging around :)